operations/bills.md

```diff
@@ -107,7 +107,15 @@ Returns all bills, optionally filtered by customers, identifiers and other filte
                 "Value": {
                     "ItalianFiscalCode": null
                 }
-            }
+            },
+            "EnterpriseData": {
+                AdditionalTaxIdentifier: null
+                CompanyName: null
+                BankAccount: null
+                BankName: null
+                Iban: null
+                Bic: null
```

Comment:
We could probably come up with some real-looking examples. right?

Reply:
This data is as general as the notes. There is no format for them it is just a string value. If I would add some examples I would think that this is the expected format for this value. I am not saying no for examples, rather I don't understand why.